resources/cluster-essentials/files/crd-applications.applicationconnector.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
```

Comment:
All the other CRDs were copied to the files directory and suppose to be installed only on upgrade. As I understand it was done so that Helm would not delete them, why is that not the case with this CRD as this one has been moved?